app/controllers/public/index.js

```diff
@@ -136,7 +136,7 @@ export default class IndexController extends Controller {
         .catch(async e => {
           console.error('Error while saving order', e);
           try {
-            await Promise.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
+            await RSVP.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['RSVP' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5581106)

Reply:
Sir what I can do for it any Suggestions?